app/liquid_tags/github_tag.rb

```diff
@@ -1,9 +1,9 @@
 class GithubTag < LiquidTagBase
-  def initialize(tag_name, link, tokens)
+  def initialize(tag_name, link, parse_context)
     super
     @tag_name = tag_name
     @link = link
-    @tokens = tokens
+    @parse_context = parse_context
```

Comment:
i think this can be removed as if changing the name didn't break anything it means it's not used by the view or the tag